### DISCUSSION TITLE
A Flutter App?

### DISCUSSION THREAD

**Discussion by dhruvkelawala** [benawad/dogehouse#833] [Ideas]
I see you guys already have a React Native app (pilaf) but I was interested in working on a Flutter App for the same project.

**Comment by Banik1103** [comment]
I'm a Flutter dev and would be very happy to work on that.

**Comment by janhendrikf** [comment]
I tried it out and made a little app.
[https://github.com/janhendrikf/dogehouse_flutter](https://github.com/janhendrikf/dogehouse_flutter)

**Comment by dhruvkelawala** [comment]
If we can get a team of 3-5 people atleast and if @benawad approves... We can work on it from scratch.

**Reply by yk-sgr** [reply]
I also would be available

**Comment by Banik1103** [comment]
I have created a group on Discord. For those of you who are interested and want to join, add me on Discord and I'll add you to the group. My Discord tag: Banik#1103

**Reply by dhruvkelawala** [reply]
Shouldn't we wait for @benawad approval? Or are we just planning to make an unofficial client?

**Reply by benawad** [reply]
there will not be an official flutter app but unofficial clients are welcome

**Reply by dhruvkelawala** [reply]
@Banik1103 I have sent you a request on Discord

**Reply by Banik1103** [reply]
Accepted!

**Comment by Banik1103** [comment]
As Ben said, there won't be an official Flutter client, but he created a GitHub issue where all clients are linked: https://github.com/benawad/dogehouse/issues/988
